### ISSUE TITLE
a number in front of each achievement

### ISSUE THREAD

**Issue by Claygame** [pokeclicker-dev/pokeclicker#74]
its just a number in front of the achievement text, this will make it easier for people to talk about them and also to know what achievement they just completed in the list

**Comment by RedSparr0w** [comment]
The main problem with this is when achievements are added/removed, it may change the order, and then the numbers wouldn't reference correctly.

Best way to reference an achievement to somebody else, would be by its name and maybe even page number.

**Comment by Claygame** [comment]
i dont think it will be a problem, it will hardly ever happen that an achievement is removed and as per when new achievements are added, their number will move from the current number onward.
Trust me, a lot of people get crazy motivation from seeing hard to achieve achievements and being able to discuss and talk and boast about them easily will help a lot with keeping community active
